### PR TITLE
Add integration test for started/finished container time

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
This adds an integration test to verify the container start/finished time.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
code LGTM, though I'm curious why we're making this change

**Comment by saschagrunert** [comment]
> code LGTM, though I'm curious why we're making this change

It more aligns to dockershim and would make a transition easier, if users parse back the value into golang and call `IsZero` on the time to verify if the field is set or not.

**Comment by haircommander** [comment]
SGTM :)
/approve

**Review by giuseppe — APPROVED** [review]
/lgtm

File: server/container_status.go [hunk 1]
```diff
@@ -65,23 +65,17 @@ func (s *Server) ContainerStatus(ctx context.Context, req *types.ContainerStatus
 		cState = c.State()
 	}
 
-	created := c.CreatedAt().UnixNano()
+	resp.Status.CreatedAt = cState.Created.UnixNano()
+	resp.Status.StartedAt = cState.Started.UnixNano()
+	resp.Status.FinishedAt = cState.Finished.UnixNano()
```
Comment (Iceber) [inline server/container_status.go:70]: It looks like the startedAt and finishedAt timestamp might be `-6795364578871345152`(`time.Time{}.UnixNano()`)
https://github.com/cri-o/cri-o/blob/f47aeb6bf10cc62b4b5af2283fa507ddc5242191/vendor/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto#L925-L938
`time.Unix(0, -6795364578871345152).IsZero()` is false

Comment (Iceber) [reply server/container_status.go:70]: The current code should be correct

Comment (saschagrunert) [reply server/container_status.go:70]: Ah, yeah thank you for the wrap-up. This was a bit confusing.

Comment (saschagrunert) [reply server/container_status.go]: Let's keep the integration test.

**Comment by saschagrunert** [comment]
The overall logic already was fine as @Iceber stated. Anyhow, let's keep the integration test case :)

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests.

**Comment by haircommander** [comment]
@saschagrunert still want the hold?

**Pull request merged**